ui/admin/app/components/form/target/add-credential-sources/index.js

```diff
@@ -0,0 +1,68 @@
+import Component from '@glimmer/component';
+import { computed, action } from '@ember/object';
+import { A } from '@ember/array';
+
+export default class FormTargetAddCredentialLibrariesIndexComponent extends Component {
```

Comment:
I don't think this affects anything but for clarity the class name should be renamed to `FormTargetAddCredentialSourcesIndexComponent`

Reply:
Good catch. Fixed.